frontend/src/views/web/landing.html

```diff
@@ -68,8 +68,8 @@ <h4 class="grad-span">
                             <p>
                                 <span class="w-500 text-light-black">Status: </span> <span class="w-500"> In Progress</span>
                                 <span class="right">
-                                    <a ng-show="isAuth" class="view-more" ui-sref="web.challenge-main.challenge-page({challengeId:main.featured-challenge/Challenge.id})">view more <i class="fa fa-external-link"></i></a>
-                                    <a ng-show="!isAuth" class="view-more" ui-sref="featured-challenge-page({challengeId:main.featuredChallenge.id})">view more <i class="fa fa-external-link"></i></a>
+                                    <a ng-show="isAuth" class="view-more" ui-sref="web.challenge-main.challenge-page({challengeId:main.featuredChallenge.id})">view more <i class="fa fa-external-link"></i></a>
+                                    <a ng-show="!isAuth" class="view-more" ui-sref="auth.login">view more <i class="fa fa-external-link"></i></a>
```

Comment:
Don't change login based route as this has done intentionally to give access to non signed in user for challenge pages @viditjain08

Reply:
@aka-jain, I get your point, but then there would be no point for having ng-if. So, I thought it would have been better to route to login page

Reply:
There is a reason to do this. So basically when user is logged in then he/she should redirect to dashboard so to minimize the number of clicks and navigation. @viditjain08